src/System/IO/Streams/SequenceId.hs

```diff
@@ -19,23 +20,28 @@ import qualified System.IO.Streams   as Streams
 -- Example:
 --
 -- @
--- ghci> 'System.IO.Streams.fromList' [1..10 :: 'Data.SequenceId.SequenceId'] >>= 'sequenceIdInputStream' 0 'id' ('fail' . 'show') >>= 'System.IO.Streams.toList'
--- [1,2,3,4,5,6,7,8,9,10]
---
--- ghci> 'System.IO.Streams.fromList' [5..10 :: 'Data.SequenceId.SequenceId'] >>= 'sequenceIdInputStream' 0 'id' ('fail' . 'show') >>= 'System.IO.Streams.toList'
--- *** Exception: user error ('Data.SequenceId.SequenceIdError' {errType = 'Data.SequenceId.SequenceIdDropped', lastSeqId = 0, currSeqId = 5})
---
--- ghci> 'System.IO.Streams.fromList' [1..10 :: 'Data.SequenceId.SequenceId'] >>= 'sequenceIdInputStream' 5 'id' ('fail' . 'show') >>= 'System.IO.Streams.toList'
--- *** Exception: user error ('Data.SequenceId.SequenceIdError' {errType = 'Data.SequenceId.SequenceIdDuplicated', lastSeqId = 5, currSeqId = 1})
+-- ghci> import qualified System.IO.Streams as Streams
+-- ghci> is <- Streams.fromList [1..10::Int]
```

Comment:
Please add single quotes around all the functions like the previous functions. That creates links in the haddock. Checkout the documentation here:

https://hackage.haskell.org/package/seqid-streams-0.7.2/docs/System-IO-Streams-SequenceId.html

It's quite nice to have that documentation. You might have to fully qualify the things in the code, but it'll come out clean in the docs.

Reply:
that's pretty cool, I'll update!